src/components/fields/switch/types.ts

```diff
@@ -0,0 +1,12 @@
+import { IFrontendEngineBaseFieldJsonSchema, TComponentOmitProps } from "../../frontend-engine";
+import { ToggleProps } from "@lifesg/react-design-system/toggle";
+
+type TCustomOptions = {
+	border?: boolean | undefined;
+};
+
+export interface ISwitchSchema<V = boolean>
```

Comment:
leave `V` as undefined
```suggestion
export interface ISwitchSchema<V = undefined>
```

Reply:
done